### ISSUE TITLE
After the build the app shows 404 error

### ISSUE THREAD

**Issue by technicusinfotech** [phonegap/build#400]
hello,

i got a problem,the issue is that my app shows great in the phonegap developer app  but when  build it its give me 404 error when the aap launches. and i m new in phonegap so tell me where i m lacking.

thanks

**Comment by willbittner** [comment]
Yeah I have been getting the same thing. I believe it is hydration. Try without hydration. I had the same issue. It is resolved on android , but I am still working on isolating it.

**Comment by technicusinfotech** [comment]
No this error is due to multiple index.html filevin your project so please check all the folders and that no index.html exist

**Comment by willbittner** [comment]
Sure - but that is a huge problem - where does it stop searching -  a large number of bower packages have a index.html. Also, there is no indication, within reason, that any sort of structure is required. Here is the first thing on google I read when I was researching the possibility this was it:
[Phone Gap Community](http://community.phonegap.com/nitobi/topics/phonegap-folder-structure-for-build)  What is this requirement constricted to?  I would be more than happy to build checks for these things if you can point me to which code base your build service is located in. 

I was also experiencing 404 on an included .js or .css file that was different each time I built - seems to be narrowed to hydration. I would be more than glad to fix both of these issues. 

Where is the build repo?

**Comment by technicusinfotech** [comment]
Yeah but when I faced the same then that time multiple index problem is there

**Comment by willbittner** [comment]
Ah gotcha - yeah - I believe they search for the files needed with grep, regex, etc. Hydration it looks like they incompletely load some files, different ones with sequential builds with no changes. Sounds like a timing bug- that would likely cause the error. I wish I could find the repo - it will take me less time to fix it than I spent figuring it out :-) Do you know if they released the code responsible for remote building?
